uber/models/department.py

```diff
@@ -407,6 +416,38 @@ def hours(self):
     def end_time(self):
         return self.start_time + timedelta(hours=self.duration)
 
+    def working_limit_ok(self, attendee):
+        attendee_hour_map = attendee.hour_map
+        hours_worked = self.duration
+        working_hour_limit = self.department_max_consecutive_hours
+        if working_hour_limit == 0:
+            working_hour_limit = 1000 # just default to something large
```

Comment:
Just to clarify, the reason you don't just `return True` here is that while this department might not have a working limit, their adjacent shift in another department would, right?